nixos/modules/flyingcircus/roles/graylog.nix

```diff
@@ -282,6 +288,12 @@ in
               proxy_set_header X-Graylog-Server-URL https://${cfg.publicFrontend.hostName}:443/api;
               proxy_pass http://${listenOn}:${toString glAPIHAPort};
           }
+
+          location /admin {
+              auth_basic "FCIO user";
+              auth_basic_user_file "/etc/local/nginx/htpasswd_fcio_users";
+              proxy_pass http://${listenOn}:${toString glAPIHAPort};
+          }
```

Comment:
Lots of duplicated lines - should probably be extracted into `let` clause

Reply:
Generally yes, and it was not readable. The whole http or https thingy makes it difficult. Maybe we should switch the listen statements, not the other stuff.